browser-interface/packages/renderer-protocol/services/crdtService.ts

```diff
@@ -1,9 +0,0 @@
-import { RpcClientPort } from '@dcl/rpc'
-import * as codegen from '@dcl/rpc/dist/codegen'
-import { CRDTServiceDefinition } from '@dcl/protocol/out-ts/decentraland/renderer/renderer_services/crdt.gen'
-
-export function registerCRDTService<Context>(
```

Comment:
@menduz Is it true that this is not used? It was automatically reported by a tool, but unclear if it's picked up by some automated tooling.